File: 1.py
# http://codeforces.com/problemset/problem/1/A

li = list(map(int, input().split()))

n = li[0]
m = li[1]
a = li[2]

x = int((n+a-1)/a)
y = int((m+a-1)/a)

answer = x*y

print(answer)

File: 1.cpp
// http://codeforces.com/problemset/problem/1/A

#include <iostream>
using namespace std;

int main()
{
  long long n, m, a;

  cin>>n>>m>>a;

  long long result = ((int)(n+a-1)/a)*((int)(m+a-1)/a);

  cout<<result;

  return 0;
}